docs/CONTRIBUTING.md

```diff
@@ -10,55 +10,68 @@ This allows us to consider and process your contribution as quickly and smoothly
 
-* **Feature Description:** Please describe the feature you would like us to change like it is in SORMAS right now. If it is about a relatively big or a very general feature, naming it briefly might be enough. If it is about a very specific detail of said feature, please try to be more specific.
-* **Problem Description:** Tell us why you want us to change the feature and what you believe is wrong or could be improved about the way it is currently designed.
-* **Proposed Change:** Describe in as much detail as possible how you would like us to change the feature and what the expected outcome of your request should look like.
-* **Possible Alternatives:** If possible, provide alternative solutions in case your proposed change can not be implemented for any reason. This is generally *optional* but might be requested at a later point in time.
+* **Problem Description:** Tell us why you want us to changeor extend an existing feature and what you believe is wrong or could be improved about the way it is currently designed.
+* **Proposed Change:** Describe in as much detail as possible how you would like us to change the feature and what the expected outcome of your request should look like. If possible, provide alternative solutions in case your proposed change can not be implemented in the way you outlined it for any reason.
+* **Acceptance Criteria:** Describes what conditions should apply to the requested change. These can be functional or non-functional requirements, that are usually specified by the developers.
+* **Implementation Details:** While refining an issue, a developer usually gives hints what to consider or change in the source code.
 * **Additional Information:** If there is anything else you want to add to your request, you can put it here. *Optional*.
 
 ### Feature Request
 
 These types of issue cover everything that involves adding new features to SORMAS. This includes both very large additions like completely new app sections, but also smaller ones like adding a new field to an existing form.
 
-* **Situation Description:** Please describe why your proposed feature is required, why SORMAS in its current state is not able to do what you want it to do, and in which way it would benefit from your feature being implemented.
-* **Feature Description:** Describe in as much detail as possible how your proposed feature should look like and what it should do.
-* **Possible Alternatives:** If possible, provide alternative solutions in case your proposed feature can not be implemented in the way you outlined it for any reason. This is generally *optional* but might be requested at a later point in time.
+* **Feature Description:** Please describe why your proposed feature is required, why SORMAS in its current state is not able to do what you want it to do, and in which way it would benefit from your feature being implemented.
```

Comment:
what you want it to do. How would you benefit from this feature being implemented?